components/ui/Accordion/index.tsx

```diff
@@ -23,11 +23,46 @@ export default function Accordion(props: Props): ReactElement {
   const [isExpanded, setIsExpanded] = useState(true);
   const [height, setHeight] = useState(`${content?.current?.scrollHeight}px`);
   const [loaded, setLoaded] = useState(false);
+
   const handleExpand = () => {
     setIsExpanded(!isExpanded);
     setHeight(isExpanded ? '0px' : `${content?.current?.scrollHeight}px`);
   };
   const svgClasses = classNames('w-3 h-3 fill-current mb-1 mr-2');
+
+  useEffect(() => {
+    const buttons = window?.document.querySelectorAll(
+      '.showExternalVideoButton'
+    );
+    const container = window?.document.getElementById(id + 'container')!;
+
+    const handleNewHeight = (e: Event) => {
```

Comment:
You can extract this function outside of the component like with what we do on session desktop 👍